File: demo/src/main/java/kaufland/com/demo/entity/UserComment.java
package kaufland.com.demo.entity;

import kaufland.com.coachbasebinderapi.MapWrapper;
import kaufland.com.coachbasebinderapi.Default;
import kaufland.com.coachbasebinderapi.Field;

@MapWrapper
public class UserComment {

    @Field(value = "comment")
    private String comment;

    @Field("user")
    @Default("anonymous")
    private String userName;
}

File: demo/src/main/java/kaufland/com/demo/entity/UserComment.kt
package kaufland.com.demo.entity

import kaufland.com.coachbasebinderapi.Fields
import kaufland.com.coachbasebinderapi.MapWrapper
import kaufland.com.coachbasebinderapi.Field

@MapWrapper
@Fields(
    Field(name = "comment", type = String::class),
    Field(name = "user", type = String::class, defaultValue = "anonymous"),
    Field("age", type = Int::class, defaultValue = "0")
)
open class UserComment
